roles/ezproxy/tasks/main.yml

```diff
@@ -133,26 +119,6 @@
     job: "/usr/bin/geoipupdate"
   when: running_on_server
 
-- name: create base config file
```

Comment:
Right now capistrano won't handle these two steps for config.txt.tmp or user.txt.tmp. I will have to check into to see if capistrano can copy files to a directory that resides outside of the capistrano directory tree.